### PR TITLE
Add platform doc link to the platforms empty message

### PR DESCRIPTION
**Pull request by lgalis**
Add platform doc link to the platforms empty message. Remove Go to sources button.
Screenshot from 2021-06-14 08-46-43

### CODE REVIEW

**Review by eclarizio — APPROVED** [review]
LGTM 👍

File: src/utilities/constants.ts [hunk 1]
```diff
@@ -32,3 +32,5 @@ export const AFTER_TYPE = 'after';
 export const RETURN_TYPE = 'return';
 
 export const MAX_RETRY_LIMIT = 3;
+export const PLATFORMS_DOC_URL =
+  'https://docs.google.com/document/d/1_q4XSHyXtSzddh8maG4w8ZOtnTwebNu8a2XdoTlfCWQ/edit#heading=h.m5qd0ouuw2ek';
```
[the remark this answers is not in the document]
Comment (cbudz) [reply src/utilities/constants.ts:36]: @eclarizio @lgalis here's the updated link: https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.0-ea/html-single/red_hat_ansible_automation_platform_installation_guide/index
I has the full

**Comment by lgalis** [comment]
@eclarizio - doc link updated

**Pull request merged**